src/probnum/random_variables/_random_variable.py

```diff
@@ -10,15 +10,14 @@
 import numpy as np
 
 from probnum import utils as _utils
+from probnum.type import ArrayLikeGetitemArgType  # Argument Types
```

Comment:
Same as in the previous comment. Just replicate whatever you end up doing in src/probnum/random_variables/_dirac.py.